manifests/router/listener.pp

```diff
@@ -18,7 +18,7 @@
 #
 define qpid::router::listener(
   $ssl_profile    = undef,
-  $addr           = '0.0.0.0',
+  $host           = '0.0.0.0',
```

Comment:
I'm thinking of not renaming these initially because that requires changes in https://github.com/theforeman/puppet-foreman_proxy_content/blob/master/manifests/dispatch_router.pp while dropping the container doesn't hurt since it's not overridden.

Reply:
Yea I know, but there's other renames, for example linkRoutePattern is now linkRoute and takes different params too.

If @ehelms could pin the 3.5 RPM's to older dispatch router then we could take care of all the upcoming deprecations in one shot and make puppet-qpid 4.x ready for dispatch router 1.0 and do that in 3.6?

I can also make a smaller scope change and just fix the container for 3.5, but who knows when they'll break us again by removing the other deprecated items

Reply:
We don't have to rename the parameters if they continue to accept the same values. It'll mean we need to do a 4.0 and a 5.0 release but that can be worth it.

Reply:
I guess, it's nicer when things match the qpid docs.  I'll open a smaller PR with the minimal changes for 1.0.